x/bank/module.go

```diff
@@ -195,3 +203,48 @@ func (am AppModule) WeightedOperations(simState module.SimulationState) []simtyp
 		simState.AppParams, simState.Cdc, am.accountKeeper, am.keeper,
 	)
 }
+
+// App Wiring
+
+func init() {
+	appmodule.Register(&modulev1.Module{},
+		appmodule.Provide(
+			provideModuleBasic,
+			provideModule))
+}
+
+func provideModuleBasic() runtime.AppModuleBasicWrapper {
+	return runtime.WrapAppModuleBasic(AppModuleBasic{})
+}
+
+type bankInputs struct {
+	depinject.In
+
+	Config        *modulev1.Module
+	AccountKeeper types.AccountKeeper `key:"cosmos.auth.module.v1.AccountKeeper"`
```

Comment:
```suggestion
	AccountKeeper types.AccountKeeper `key:"cosmos.auth.v1.AccountKeeper"`
```

maybe we can skip `module` in the name?